test/minishift/test_suite.go

```diff
@@ -92,16 +93,21 @@ func prepareConfig(t *testing.T) (*configuration.Data, func()) {
 	return config, reset
 }
 
-func VerifyObjectsPresence(t *testing.T, mappedObjects map[string]environment.Objects, options openshift.ApplyOptions, version string) {
+func VerifyObjectsPresence(t *testing.T, mappedObjects map[string]environment.Objects, options openshift.ApplyOptions, version string, required bool) {
 	size := 0
 	for _, objects := range mappedObjects {
 		size += len(objects)
 	}
 	errorChan := make(chan error, size)
 	defer func() {
 		close(errorChan)
+		errWasFound := false
 		for err := range errorChan {
 			assert.NoError(t, err)
```

Comment:
same here: use `require.NoError`

Reply:
check the parameter `required` ;-). if this evaluation is part of `// given` section, then if there was any error found, then the setup is stopped: https://github.com/fabric8-services/fabric8-tenant/pull/680/files/ff391be402df912216ddd888fc1e9eb023fd3bee#diff-e526609efd4c9f26c792830751f918e4R110 as a benefit we will get the context of all errors that occured during the setup.

Reply:
so why don't you just replace line 106 with something like:
```
if required {
  require.NoError(t, err)
}
``` 
and otherwise, just ignore the error (or log it, if needed)

Reply:
ok, got it.